### DISCUSSION TITLE
how do i prevent opening a new terminal window by clicking a windows shortcut?

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#5957] [Q&A] [answered]
i followed the instructions [here](https://qutebrowser.org/INSTALL.html#tox) to build qb from source on windows, created a shortcut on desktop (right-click->new->shortcut), entered `path\to\.venv\Scripts\python.exe`, then i edited the shortcut properties, and added `-m qutebrowser` to the target, now everything works, but it opens a new terminal window and only then qutebrowser itself

**Comment by ghost** [comment, marked as answer]
oh, i just tried `pythonw.exe` instead of `python.exe` and that solved my problem!

**Comment by The-Compiler** [comment]
Note you can also run `tox -e build-release` to build a "proper" release (with installers and everything).
